### DISCUSSION TITLE
Runelite side bar  bug (ALIENWARE)

### DISCUSSION THREAD

**Discussion by GHO5TTTT** [runelite/runelite#18205] [Q&A]
For my Alienware users... got the sidebar glitch again after years I tried everything YouTube has said and it's still glitching (only started happening today) I've seen suggestions to remove the command center but that sounds a bit scary any thoughts on what to do?

**Comment by cwjoshuak** [comment]
have you tried [disabling hardware acceleration](https://github.com/runelite/runelite/wiki/Disable-Hardware-Acceleration)?

**Reply by GHO5TTTT** [reply]
I did but method 2 is confusing not sure if I did it right but I typed the mode off thing into it and still nothing today is the first day I'm seeing the glitch it's been well over a year since I've seen it so I'm not sure if Nvidia and Alienware control center is the issue and if it is ACC controls my fans and bios updates so if I remove that my laptop would take a dump I think
